### ISSUE TITLE
Are there any problems about these tow lines?

### ISSUE THREAD

**Issue by XiangqianMa** [antspy/quantized_distillation#14]
When I tried to run DISTILLED_MODEL in cifar10_wideResNet.py, I found maybe there are some errors with these two lines (in cnn_models/help_fun.py):
```
index_distillation_loss = torch.arange(0, outputs.size(0))[mask_distillation_loss.view(-1, 1)].long()
inverse_idx_distill_loss = torch.arange(0, outputs.size(0))[1-mask_distillation_loss.view(-1, 1)].long()
```
The size of the tensor which produced from ``torch.arange(0, outputs.size(0))``  is [outputs.size(0)], but it's [batch_size, 1] of ``mask_distillation_loss``. So maybe there are some errors with these two lines.
It will be appreciate if I receive your reply.

**Comment by antspy** [comment]
Hi, 

Sorry for the late reply. 
Are you using pytorch 0.3.1? That part of the code was used for other experiments not used in the paper, so it's possible that I broke something when uploading the paper code on github.

**Comment by XiangqianMa** [comment]
Thanks for your reply. My pytorch version is 1.0. It is because of the mismatched index between these two tensors. Changing the mask tensor can fix it.

**Comment by antspy** [comment]
Thanks for the suggestion! Is this issue present in pytorch 0.3.1 as well? I know pytorch 1.0 handles lots of things differently, so the mismatch could be caused from that. 

Let me add that the code is not intended to work on pytorch versions after 0.3.1, as you will likely find multiple issues like this. Unfortunately I don’t have the time for a port this repo to pytorch 1.0.

**Comment by XiangqianMa** [comment]
I haven't tried pytorch 0.3.1, maybe it could work. You're right, there are so many things changed in 1.0. (Maybe, there are something different in the ``tensor.view()`` API ).
